packages/website/src/components/CurriculumSection/CurriculumSection.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { Button } from '@coderscamp/ui/components/Button';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { VStack } from '@coderscamp/ui/components/Stack';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { SolidDownloadIcon } from '@coderscamp/ui/icons';
+
+import { TechSliding } from './TechSliding';
+
+export const CurriculumSection = () => {
+  const textSize = { base: 'md', md: 'lg' } as const;
+
+  return (
+    <Flex justify="center" align="center" direction="column" my="80px" width="100%">
+      <Typography size="4xl" weight="bold" color="gray.900" p={{ base: '32px 16px 64px' }}>
+        Czego nauczysz się na kursie?
+      </Typography>
+      <TechSliding />
+
+      <VStack maxW="min(1280px, 100%)" my="40px">
+        <Typography as="p" p={{ base: '32px 16px 64px' }} color="gray.500" size={textSize} textAlign="center">
```

Comment:
💭 According to mockups there should be no top padding (it's provided by the Stack component) and padding-bottom should be only 32px